front/src/applications/operationalStudies/views/Scenario/components/Timetable/TimetableBoardWrapper.tsx

```diff
@@ -114,6 +115,67 @@ const TimetableBoardWrapper = ({
     [selectedTimetableItemIds]
   );
 
+  const removeAndUnselectTrains = useCallback(
+    (timetableItemIds: TimetableItemId[]) => {
+      removeTimetableItems(timetableItemIds);
+      setSelectedTimetableItemIds([]);
+    },
+    [removeTimetableItems, setSelectedTimetableItemIds]
+  );
+
+  const ctrlC = useCallback(
+    (event: KeyboardEvent) => {
+      if (!event.repeat && event.ctrlKey && event.key === 'c') {
+        copyTimetableItemsToClipboard(selectedTimetableItemIds, timetableItems);
+      }
+    },
+    [selectedTimetableItemIds, timetableItems]
+  );
+
+  const ctrlV = useCallback(
+    async (event: KeyboardEvent) => {
+      if (!event.repeat && event.ctrlKey && event.key === 'v') {
```

Comment:
Could we use the `paste` event instead of catching a specific combination of keypresses?